zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -74,22 +76,45 @@ const char * zstdVersion() { return ZSTD_VERSION_STRING; }
 ZEXTERN const char * ZEXPORT z_zlibVersion OF((void)) { return zlibVersion();  }
 
 
+void* ZWRAP_allocFunction(void* opaque, size_t size)
```

Comment:
There is no declaration of ZWRAP_allocFunction so it's not accessible from outside.
It will be not inlined as it used as:
```
ZSTD_customMem ZWRAP_customMem = { ZWRAP_allocFunction, ZWRAP_freeFunction, &zwd->allocFunc };
```
Using static can be only helpful to remove it from symbols list.